CBLClient/Database.py

```diff
@@ -121,10 +121,8 @@ def updateDocuments(self, database, documents):
         args.setDictionary("documents", documents)
         return self._client.invokeMethod("database_updateDocuments", args)
 
-    def updateDocument(self, database, data, doc_id, attachments_name=None):
+    def updateDocument(self, database, data, doc_id):
```

Comment:
what happens to the case in which we want to update attachment?

Reply:
we are using blobs now, there is no attachment in CBL. When you update document, data should have image added and image comes along with data. There is no need of attachment parameter as we don't use attachments. I made sure it is not used in any tests.

Reply:
If we don't have a test which doesn't update attachment, then we should write one. This is miss on our part that we are not testing anywhere, how update behaves if we just update attachment but not content

Reply:
already added @hemant, that is the whole reason for adding blob support as test is failing.